invenio/modules/communities/helpers.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Module provides helper functions for communities views."""
+
+import os
+
+from invenio.base.globals import cfg
+
+
+def save_and_validate_logo(logo, filename, prev_ext=None):
+    """Validate if communities logo is in limit size and save it."""
+    base_path = os.path.join(cfg['COLLECT_STATIC_ROOT'], 'user')
```

Comment:
Should we use `Documents` or at least pyfilesystem to allow different storages? Using `COLLECT_STATIC_ROOT` is not correct because it requires shared filesystem among web nodes.